psq/queue.py

```diff
@@ -34,11 +34,15 @@
 
 class Queue(object):
     def __init__(self, pubsub, name='default', storage=None,
-                 extra_context=None):
-        _check_for_thread_safety(pubsub)
-        self.pubsub = pubsub
+                 extra_context=None, async=True):
+        self.async = async
```

Comment:
Oh, and I don't think it'll collide with python3.5 (doesn't in the unit tests, at least). I think python's keyword is in fact `async def` (always in compound). 

It's a bit scary that `async` is colored differently in some editors, though. Then again, RQ does this the same way, so I think we should do that, too.